src/analyze/analyze-security.c

```diff
@@ -1834,19 +1806,22 @@ static int property_read_device_allow(
         return sd_bus_message_exit_container(m);
 }
 
-static int acquire_security_info(sd_bus *bus, const char *name, struct security_info *info, AnalyzeSecurityFlags flags) {
+static int acquire_security_info(sd_bus *bus,
+                                 const char *name,
+                                 struct security_info *info,
+                                 AnalyzeSecurityFlags flags) {
 
         static const struct bus_properties_map security_map[] = {
                 { "AmbientCapabilities",     "t",       NULL,                                    offsetof(struct security_info, ambient_capabilities)      },
                 { "CapabilityBoundingSet",   "t",       NULL,                                    offsetof(struct security_info, capability_bounding_set)   },
                 { "DefaultDependencies",     "b",       NULL,                                    offsetof(struct security_info, default_dependencies)      },
                 { "Delegate",                "b",       NULL,                                    offsetof(struct security_info, delegate)                  },
-                { "DeviceAllow",             "a(ss)",   property_read_device_allow,              0                                                         },
+                { "DeviceAllow",             "a(ss)",   property_read_device_allow,              0                             },
```

Comment:
hmm, table got corrupted?

Reply:
This was intended, since the last parameter of that line is omitted. 
But i can add empty spaces to keep the right alignment of the closing brace, if desired?

Reply:
hmm? the last parameter is not ommitted, it's set explicitly to 0 though?

Reply:
sorry, got confused, i thought these were two separate parameters and the parameter count changed in the lines, but the offsetof(...) is actually one parameter. I will fix this...

Reply:
fixed here: https://github.com/systemd/systemd/pull/11690/commits/5ea5312ff3b97a1c31e7c9b55678df3243ca5284